3.1-degrading-multisig-case-study.ipynb

```diff
@@ -395,7 +427,7 @@
     "spending_tx.vin = [spending_tx_in]\n",
     "spending_tx.vout = [dest_output]\n",
     "\n",
-    "sighash = TaprootSignatureHash(spending_tx, [output], SIGHASH_ALL_TAPROOT, 0, scriptpath=True, tapscript=tapscript_2a.script)\n",
+    "sighash = TaprootSignatureHash(spending_tx, [output], SIGHASH_ALL_TAPROOT, 0, scriptpath=True, script=tapscript_2a.script)\n",
```

Comment:
17 lines below here and ~100 lines below here In 9f49b34872791fd4f689fffc08fb523975dd0beb:
The `testmempoolaccept` assertion should have `'reject-reason' : 'non-BIP68-final'` instead of `'reject-reason' : '64: non-BIP68-final'`.

Reply:
Addressed (idk why but I can't resolve comments haha)